lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb

```diff
@@ -216,15 +435,15 @@ def self.update_version(version, section)
                 end
               else
                 if (version_updated < 2)
-                  if (line.include? "versionName") and (!line.include? "\"versionName\"") and (!line.include?"PversionName")
+                  if line.include?("versionName") && !line.include?('"versionName"') && !line.include?("PversionName")
                     version_name = line.split(' ')[1].tr('\"', '')
-                    line.replace line.sub(version_name, version[VERSION_NAME].to_s)
+                    line.sub!(version_name, version[VERSION_NAME].to_s)
```

Comment:
While `String#sub` returns a new string in ruby, `String#sub!` modifies the string in-place. So more idiomatic and performant than calling `replace` on the same string with the returned value like before.